linera-execution/src/runtime.rs

```diff
@@ -1385,6 +1424,33 @@ impl ServiceSyncRuntime {
         }
     }
 
+    pub(crate) fn preload_service(
+        &self,
+        id: UserApplicationId,
+        code: UserServiceCode,
+        description: UserApplicationDescription,
+    ) -> Result<(), ExecutionError> {
+        let this = self
+            .runtime
+            .0
+            .as_ref()
+            .expect("services shouldn't be preloaded while the runtime is being dropped");
+        let runtime_handle = this.clone();
+        let mut this_guard = this.inner();
+
+        if let std::collections::hash_map::Entry::Vacant(e) =
```

Comment:
Nit: Could we import `std::collections::hash_map` to make this a bit easier to read?.